esmvalcore/cmor/_fixes/emac/emac.py

```diff
@@ -211,6 +211,30 @@ def fix_metadata(self, cubes):
         return CubeList([cube])
 
 
+class Prodlnox(EmacFix):
+    """Fixes for ``prodlnox``."""
+
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        cube = ((
+            self.get_cube(cubes, var_name=['NOxcg_cav', 'NOxcg_ave',
+                                           'NOxcg']).collapsed(
+                                               ['longitude', 'latitude'],
+                                               iris.analysis.SUM,
+                                               weights=None) +
+            self.get_cube(cubes, var_name=['NOxic_cav', 'NOxic_ave',
+                                           'NOxic']).collapsed(
+                                               ['longitude', 'latitude'],
+                                               iris.analysis.SUM,
+                                               weights=None)) /
+                self.get_cube(cubes, var_name=['dt'])) * 365 * 24 * 3600 / 1e9
+
+        cube.var_name = self.vardef.short_name
+        cube.units = 'Tg a-1'
```

Comment:
```suggestion
                self.get_cube(cubes, var_name=['dt'])
        )
        cube.convert_units('Tg yr-1')
        cube.var_name = self.vardef.short_name
```

This should be safer and easier to read. Also note that the unit `a` is interpreted as are (100m2), not year.

Reply:
Thank you for this comment. I was not aware of that. Do you know, if the cf unit converter is able to convert kg s-1 to Tg yr-1? I think it makes more sense to define the variable in SI units and do the unit conversion by the preprocessor.

Reply:
Yes, it is (I just tested it). It's also fine for me to use the SI, but if you'd like you can keep it like that. We have lots of custom tables that do not use SI units.

Reply:
It seems like converting the units at that point does not work, as it assumes some strange untis for the cubes of NOxcg and NOxic. They are defined with `kg(N)` in MESSy and that might be a problem as `N` is assumed as Newton by the conversion.
I suppose I have to overwrite them.